tests/integrations/config/providers.py

```diff
@@ -47,4 +49,6 @@
     ValidationProvider,
     MyTestPackageProvider,
     AppProvider,
+    ORMProvider,
+    InternalORMProvider,
```

Comment:
I don't like 2 different ORM Providers. we should put all of Masonite ORM's commands inside 1 Masonite provider inside Masonite core and modify the commands to accept inputs for anything diff

Reply:
I edited the PR at the same time there are no different providers anymore.